rpcs3/Emu/system_config.h

```diff
@@ -166,7 +166,7 @@ struct cfg_root : cfg::node
 		cfg::_int<1, 8> consecutive_frames_to_skip{ this, "Consecutive Frames To Skip", 1, true};
 		cfg::_int<50, 800> resolution_scale_percent{ this, "Resolution Scale", 100 };
 		cfg::uint<0, 16> anisotropic_level_override{ this, "Anisotropic Filter Override", 0, true };
-		cfg::_int<-16, 16> texture_lod_bias{ this, "Texture LOD Bias Addend", 0, true };
```

Comment:
Lod bias affects the computed mip level. The mip levels are a function of log2(dimension) where dimension is the size of the texture. PS3 supports upto 4096x4096 size images which is only 11 possible levels, so 16 was already more than enough. Just leave it as -16 to +16 which technically covers textures upto 32k size (log2(32768) + 1).

Reply:
I want it to be able to shift the value towards the limits regardless to the value passed by the game. For example if the game already passes +16 because some mistake of calculation this can shift it and even make it negative now.